langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/AnthropicMapper.java

```diff
@@ -44,52 +114,29 @@ static String toAnthropicSystemPrompt(List<ChatMessage> messages) {
         }
     }
 
-    private static AnthropicRole toAnthropicRole(ChatMessageType chatMessageType) {
-        switch (chatMessageType) {
-            case AI:
-                return AnthropicRole.ASSISTANT;
-            case USER:
-                return AnthropicRole.USER;
-            default:
-                throw new IllegalArgumentException("Unknown chat message type: " + chatMessageType);
-        }
-    }
-
-    private static Object toAnthropicContent(ChatMessage message) {
-        if (message instanceof AiMessage) {
-            AiMessage aiMessage = (AiMessage) message;
-            return aiMessage.text();
-        } else if (message instanceof UserMessage) {
-            UserMessage userMessage = (UserMessage) message;
-            return userMessage.contents().stream()
-                    .map(content -> {
-                        if (content instanceof TextContent) {
-                            return new AnthropicTextContent(((TextContent) content).text());
-                        } else if (content instanceof ImageContent) {
-                            Image image = ((ImageContent) content).image();
-                            if (image.url() != null) {
-                                throw illegalArgument("Anthropic does not support images as URLs, " +
-                                        "only as Base64-encoded strings");
-                            }
-                            return new AnthropicImageContent(
-                                    ensureNotBlank(image.mimeType(), "mimeType"),
-                                    ensureNotBlank(image.base64Data(), "base64Data")
-                            );
-                        } else {
-                            throw illegalArgument("Unknown content type: " + content);
-                        }
-                    }).collect(toList());
-        } else {
-            throw new IllegalArgumentException("Unknown message type: " + message.type());
-        }
-    }
-
     public static AiMessage toAiMessage(List<AnthropicContent> contents) {
+
         String text = contents.stream()
                 .filter(content -> "text".equals(content.type))
                 .map(content -> content.text)
                 .collect(joining("\n"));
-        return AiMessage.from(text);
+
+        List<ToolExecutionRequest> toolExecutionRequests = contents.stream()
+                .filter(content -> "tool_use".equals(content.type))
+                .map(content -> ToolExecutionRequest.builder()
+                        .id(content.id)
+                        .name(content.name)
+                        .arguments(GSON.toJson(content.input))
```

Comment:
This unfortunately breaks the cases where GSON is not used (like Quarkus which swaps the JSON handling for Jackson in the supported modules).

Reply:
Oh right! Let me fix this

Reply:
🙏🏼

Reply:
Using Gson [here](https://github.com/langchain4j/langchain4j/blob/d118c0c3b67e01bb2f159cf7a388533dac06ffad/langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/DefaultAnthropicClient.java#L168) is ok for you?

Reply:
That one never gets called in the Quarkus case (or at least I didn't see it), so it should be fine